### ISSUE TITLE
Accuracy of the evaluated model on objectron was abnormally high

### ISSUE THREAD

**Issue by nightgoodl** [UVA-Computer-Vision-Lab/ovmono3d#12]
Thanks for presenting such an excellent job, but I'm having some problems reproducing ovmono3d.
I used only the objectron dataset for training and evaluation, and I realized that objectron doesn't exist to be tested using the NOVEL category, so only the BASE category was used for testing, and then I noticed that the accuracy of the evaluated model was abnormally high, is this reasonable, and what is the reason for such a phenomenon?
 Here is the evaluation data for 10,000 iter：

<img width="1034" alt="Image" src="https://github.com/user-attachments/assets/73b20deb-5072-410d-9d82-79f3cdcad27e" />

**Comment by yaojin17** [comment]
Hello, do you mean 32.018 AP3D is too high for objectron test set? Actually, our pre-trained model can achieve 57.69 AP3D whereas Cube RCNN achieves 54.38. This is not abnormally higher than Cube RCNN.

**Comment by nightgoodl** [comment]
> Hello, do you mean 32.018 AP3D is too high for objectron test set? Actually, our pre-trained model can achieve 57.69 AP3D whereas Cube RCNN achieves 54.38. This is not abnormally higher than Cube RCNN.

 Thanks for the reply, I may have confused the paper with the actual running model, I didn't evaluate on the pre-trained model you guys gave, so is it on the objectron dataset that has better evaluation results compared to the other datasets, because I see that the AP3D written in the paper isn't this high

<img width="404" alt="Image" src="https://github.com/user-attachments/assets/44f72c2d-559c-4587-a608-edfcf2d8901f" />

**Comment by nightgoodl** [comment]
Also, after 40,000iter of training, my AP2D reached 31,AP3D reached 47, is this a normal figure?

**Comment by yaojin17** [comment]
I didn't train the model solely on objectron, so I am not clear about the specific metrics in the training process. If your settings  including learning rate, batch size, etc, are same to the config we provided, I think this performance is normal.
